### PR TITLE
Added no-worker pendant of my worker demos

### PR DESCRIPTION
**Pull request by maudnals**
Helps to showcase the performance win of using a web worker.

### CODE REVIEW

File: perf-client-side-gemma-no-worker/src/script.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+let llmInference = null;
+
+(async function () {
+  displayModelStatus(MODEL_STATUS.PREPARING);
+  document.getElementById('inferenceButton').disabled = true;
+  try {
+    const genai = await FilesetResolver.forGenAiTasks(MEDIAPIPE_WASM);
+    llmInference = await LlmInference.createFromModelPath(genai, MODEL_URL);
+    displayModelStatus(MODEL_STATUS.READY);
+    document.getElementById('inferenceButton').disabled = false;
```
[the remark this answers is not in the document]
Comment (tomayac) [reply perf-client-side-gemma-no-worker/src/script.js:15]: Throughout for all DOM references.

[the remark this answers is not in the document]
Comment (maudnals) [reply perf-client-side-gemma-no-worker/src/script.js]: Done for elements referenced several times, ty

**Review by tomayac — APPROVED** [review]
LGTM, approval.

Thanks for taking the previous nits into account.

**Comment by maudnals** [comment]
Thank you for reviewing!

**Pull request merged**